### PR TITLE
docs(pie-monorepo): DSW-000 update the pull request template

### PR DESCRIPTION
**Pull request by jamieomaguire**
## Describe your changes (can list changeset entries if preferable)
- Updates the base pull request template to include more checks and add an area for checklist items to be moved to if they are not applicable to a PR (rather than deleting and losing visibility)
- Fixes our component status tag in Storybook to use the new colour variants

## Author Checklist (complete before requesting a review, do not delete any)
- [x] I have performed a self-review of my code.
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview.

## Not-applicable Checklist items
Please move any Author checklist items that do not apply to this pull request here.
- [ ] I have added thorough tests where applicable (unit / component / visual).
- [ ] I have reviewed visual test updates properly before approving.
- [ ] If changes will affect consumers of the package, I have created a changeset entry.
- [ ] If a changeset file has been created, I have tested these changes in [PIE Aperture](https://github.com/justeattakeaway/pie-aperture/) using the `/test-aperture` command.

---

## Testing
[How do I test my changes?](https://github.com/justeattakeaway/pie/wiki/PIE-Aperture)

| Task                   | Link                             |
|------------------------|----------------------------------|
| Aperture PR            | [🔗](#) |
| NextJS 14 deployment   | [🔗](#) |
| Nuxt 3 deployment      | [🔗](#) |
| Vanilla deployment     | [🔗](#) |

## Reviewer checklists (complete before approving)
Mark items as `[-] N/A` if not applicable.

### Reviewer 1 (@maledr5)
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview.
- [-] N/A - I have verified that all acceptance criteria for this ticket have been completed.
- [-] N/A - I have reviewed the Aperture changes (if added)
- [-] N/A - If there are visual test updates, I have reviewed them.

### Reviewer 2  (@siggerzz)
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview.
- [-] N/A - I have verified that all acceptance criteria for this ticket have been completed.
- [-] N/A - I have reviewed the Aperture changes (if added)
- [-] N/A - If there are visual test updates, I have reviewed them.

### CODE REVIEW

**Review by maledr5 — APPROVED** [review]

File: apps/pie-storybook/data/tag-variants-to-statuses-map.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-    alpha: 'yellow',
-    beta: 'yellow',
-    deprecated: 'red',
-    removed: 'red',
-    stable: 'green',
+    alpha: 'brand-05',
+    beta: 'brand-05',
+    deprecated: 'error',
+    removed: 'error',
+    stable: 'success',
```
Comment (fernandofranca) [inline apps/pie-storybook/data/tag-variants-to-statuses-map.ts:8]: Was this intended to change?

**Review by fernandofranca — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -8,19 +8,36 @@ Please click the `Preview` tab and select a different PR template if your change
...
+## Author Checklist (complete before requesting a review, do not delete any)
+- [ ] I have performed a self-review of my code.
+- [ ] I have added thorough tests where applicable (unit / component / visual).
+- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview.
+- [ ] I have reviewed visual test updates properly before approving.
 - [ ] If changes will affect consumers of the package, I have created a changeset entry.
-- [ ] If a changeset file has been created, I have used the `/snapit` functionality to test my changes in a consuming application
+- [ ] If a changeset file has been created, I have tested these changes in [PIE Aperture](https://github.com/justeattakeaway/pie-aperture/) using the `/test-aperture` command.
+
+## Not-applicable Checklist items
```
Comment (ashleynolan) [inline .github/pull_request_template.md:19]: I'm guessing we'll need to add something to danger to not check this section so it doesn't moan at us?

Comment (jamieomaguire) [reply .github/pull_request_template.md:19]: yes! trying an update now - good spot

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -8,19 +8,38 @@ Please click the `Preview` tab and select a different PR template if your change
...
+
+## Testing
+[How do I test my changes?](https://github.com/justeattakeaway/pie/wiki/PIE-Aperture)
+
+| Task                   | Link                             |
+|------------------------|----------------------------------|
+| Aperture PR            | [🔗](#) |
+| NextJS 14 deployment   | [🔗](#) |
+| Nuxt 3 deployment      | [🔗](#) |
+| Vanilla deployment     | [🔗](#) |
```
Comment (siggerzz) [inline .github/pull_request_template.md:32]: Not sure if we should bother putting the deployment links here. I think we discussed a while back that we should probably update the `test-aperture` command to provide feedback once complete to provide these links

Comment (jamieomaguire) [reply .github/pull_request_template.md:32]: I can see your point - the aim of this was to make it far more intentional and less passive on the author's side. How about we add them for now and revisit once we've done the changes you suggest? Would love to pair on it! 👍

**Review by siggerzz — APPROVED** [review]

**Review by ashleynolan — COMMENTED** [review]
Tiny NIT, then LGTM

File: dangerfile.js [hunk 4]
```diff
@@ -17,17 +17,15 @@ danger.git.created_files.filter((filepath) => filepath.includes('.changeset/') &
...
 
             if (isDependabotPR) {
-                // Check if at least one of the valid changeset categories is present
                 if (numberOfCategories === 0) {
                     fail(`:memo: Your changeset doesn't include a category. Please add one of: \`${validChangesetCategories.join(', ')}\`. Filepath: \`${filepath}`);
                 } else if (!validChangesetCategories.some((cat) => changesetCategories.includes(cat))) {
                     fail(`:memo: Your changeset includes an invalid category. Please use one of: \`${validChangesetCategories.join(', ')}\`. Filepath: \`${filepath}`);
                 }
 
-                // Check that categories are followed are in the following format `[Category] - {Description}`
```
[the remark this answers is not in the document]
Comment (jamieomaguire) [reply dangerfile.js:27]: oops - yeah sure

**Review by siggerzz — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**